projects/plugins/boost/compatibility/page-optimize.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+function jetpack_boost_page_optimize_js_excludes( $default ) {
+	$exclude_list = get_option( 'page_optimize-js-exclude' );
+	if ( ! $exclude_list ) {
+		return $default;
+	}
+
+	delete_option( 'page_optimize-js-exclude' );
```

Comment:
I think we can migrate data without being destructive to external plugins.

Reply:
Updated with 37372a9999b62841096c84cda8a08a87bd4280ab.